content/issues/2021-04-14-maintenance-publick8s.md

```diff
@@ -0,0 +1,19 @@
+---
+title: AKS publick8s version update
+date: 2021-04-14T07:30:00-00:00
+resolved: true
```

Comment:
```suggestion
resolved: false
```

Reply:
That's a trick in cstate, if you put false then the notification is displayed with a red border and if set to true then it's displayed with a blue border.